linear_decoder.ipynb

```diff
@@ -1,702 +1,900 @@
 {
```

Comment:
<p>add label for train and test set and legend (quick start has this)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>